carmin-server/server/test/resources/test_execution.py

```diff
@@ -39,3 +43,94 @@ def test_get_invalid_execution(self, test_config):
             })
         error = error_from_response(response)
         assert error == EXECUTION_NOT_FOUND
+
+    def test_put_valid_execution_name_and_timeout(self, test_config,
+                                                  execution_id):
+        response = test_config.test_client.put(
+            '/executions/{}'.format(execution_id),
+            headers={"apiKey": standard_user().api_key},
+            data=json.dumps(PATCH_VALID_EXECUTION))
```

Comment:
For uniformity, we should use json_request_data utils function. I just realized it isn't used in Executions either. It does the same thing, except it was placed under a standard utils function. However, do you feel like it may be a bit overkill to have such utils function? I did it back when I wrote our first unit tests to insure uniformity, but rethinking about it, as there isn't any fancy parameter used for the moment, we could simply call json.dumps. What are you thoughts on that @louis-ver?

Reply:
I don't see the need for the `json_request_data` function. It is less explicit and does not really provide an advantage.

However, I think that `load_json_data` is useful, since the actual `json` function is a little verbose and not really helpful.

I say we remove `json_request_data`, but keep `load_json_data`